palo_alto_pan_os/help.md

```diff
@@ -33,6 +33,50 @@ The connection configuration accepts the following parameters:
 
 ### Actions
 
+#### Create Address Object
+
+This action is used to create a new address object.
```

Comment:
We need to get give some additional details here for the user. Document what we support for autodetection and document that we do not do a subnet in CIDR check with an example for illustration.

Reply:
Fixed